src/init/init.c

```diff
@@ -54,6 +54,10 @@ int master_core_init(struct sof *sof)
 	trace_point(TRACE_BOOT_SYS_POWER);
 	pm_runtime_init();
 
+	/* Turn off memory for all unused cores*/
+	for (i = 1; i < PLATFORM_CORE_COUNT; i++)
+		pm_runtime_get(CORE_MEMORY_POW, i);
```

Comment:
I think it's should be put for disabling. Get is when we want to use the resource and we need it powered up or clock gating disabled.